tests/App/AppKernel.php

```diff
@@ -71,6 +71,11 @@ public function getProjectDir(): string
         return __DIR__;
     }
 
+    /**
+     * TODO: Use RoutingConfigurator when dropping support of symfony < 5.1.
```

Comment:
see mediabundle master branch to see how it can be fixed

Reply:
👍 I've removed the type declaration